prime-router/src/main/kotlin/Element.kt

```diff
@@ -288,7 +288,19 @@ data class Element(
             }
             Type.TELEPHONE -> {
                 // normalized telephone always has 3 values national:country:extension
-                val parts = normalizedValue.split(phoneDelimiter)
+                val parts = if (normalizedValue.contains(phoneDelimiter)) {
+                    normalizedValue.split(phoneDelimiter)
+                } else {
+                    // remove parens from HL7 formatting
+                    listOf(
+                        normalizedValue
+                            .replace("(", "")
+                            .replace(")", ""),
+                        "1",    // country code
+                        ""      // extension
+                    )
+                }
+
```

Comment:
When we do the reverse direction of data in from HL7, phone number is in a different format. The HL7 serializer should, on ingest, put the phone number into the format we expect. But I wanted to put a guard around this logic because it never checks if the `:` is in the phone number or not.